pedidos/models.py

```diff
@@ -24,8 +25,7 @@ class Pedido(models.Model):
     data_resposta = models.DateField(
         "Data de resposta", db_index=True, null=True, blank=True
     )
-    orgao = models.CharField("Órgão", max_length=50, blank=True)
-    esfera = models.CharField("Esfera", max_length=20, choices=ESFERA_OPTIONS)
+    orgao = models.ForeignKey("Orgao", on_delete=models.CASCADE)
```

Comment:
```suggestion
    orgao = models.ForeignKey("Órgão", on_delete=models.PROTECT)
```
https://docs.djangoproject.com/en/3.2/ref/models/fields/